release.yaml

```diff
@@ -10,8 +10,10 @@ system-upgrade-controller:
 - 100.0.3+up0.3.2
 rancher-gatekeeper:
 - 100.1.1+up3.7.1
```

Comment:
If 100.2.0+up3.8.1 has the fix for [this issue](https://github.com/rancher/rancher/issues/37029) and the changes made in [this PR](https://github.com/rancher/charts/pull/1938), it would be best to remove 100.1.1+up3.7.1 and close that other PR so that we only release one new version of the gatekeeper chart.

Reply:
Removed the `100.1.1+up3.7.1` chart.